test/functional/test_framework/wallet.py

```diff
@@ -119,13 +122,14 @@ def _bulk_tx(self, tx, target_weight):
         """Pad a transaction with extra outputs until it reaches a target weight (or higher).
         returns the tx
         """
-        tx.vout.append(CTxOut(nValue=0, scriptPubKey=CScript([OP_RETURN, b'a'])))
+        tx.vout.append(CTxOut(nValue=0, scriptPubKey=CScript([OP_RETURN])))
         dummy_vbytes = (target_weight - tx.get_weight() + 3) // 4
-        tx.vout[-1].scriptPubKey = CScript([OP_RETURN, b'a' * dummy_vbytes])
-        # Lower bound should always be off by at most 3
+        # compensate for the compact-size encoded script length
+        dummy_vbytes -= len(ser_compact_size(dummy_vbytes)) - 1
```

Comment:
I've been trying to think why 1 is subtracted here, but could not come up with a thorough reason. Is it somehow related to excluding the already serialised length of the output script before padding?

Reply:
Thanks for the confirmation, appreciate it!